### PR TITLE
Cleanup README / Move examples

### LINKED ISSUE #370: Direct users to synthetics-demo repo
We should probably direct users to the less confusing https://github.com/elastic/synthetics-demo repo rather than the examples folder here. Nesting examples in the node package can cause weird behavior, and it's easier to maintain the examples as a separate repo. We should probably keep the folder, but add a README pointing users to the repo above.

Whether we should delete the examples or hide them somewhere obscure is a separate question.

Thoughts @vigneshshanmugam and @dominiqueclarke ?

### PR DESCRIPTION
**Pull request by andrewvc**
+ fixes #370 
This PR removes all examples, pointing people to the new synthetics-demo repo.

We should merge https://github.com/elastic/observability-docs/pull/1108 first so this doesn't break the docs.

It also cleans up the README a bit, making it more clear you must run synthetics inside the test dir. since the docs are going to be in flux for a bit it probably makes more sense to have just a single link out to the docs, even if it means our README looks a little less substantial.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ npm install -g @elastic/synthetics
 Run the suites via CLI
 
 ```
-npx @elastic/synthetics [options] [dir] [files] file
+cd my-test-dir; npx @elastic/synthetics [options] [dir] [files] file
```
Comment (dominiqueclarke) [inline README.md:18]: We have in the documentation section links to running with heartbeat. Should we consider adding a link out to running with Fleet and Elastic Agent as well?

**Review by dominiqueclarke — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (vigneshshanmugam) [inline README.md:18]: We should not have this on the main readme page. It is handled via directory option.

**Review by vigneshshanmugam — APPROVED** [review]
LGTM pending the readme change.

**Comment by vigneshshanmugam** [comment]
@andrewvc Can you rebase this with master? E2E tests were broken before and got fixed now.

**Comment by vigneshshanmugam** [comment]
Closing this PR in favor of init command and we will eventually get rid of local examples post 8.3

**Pull request closed without merging**